### ISSUE TITLE
ForgotPassword View not loading after sending email

### ISSUE THREAD

**Issue by assadnazar** [phalcon/vokuro#126]
I am working on a portal and using vokuro as a baseline. When I enter email address ForgotPassword, the email is received properly but view does not load and just the flash message on the screen on a white screen.

If i comment out,
```
$this->getDI()
       ->getMail()
       ->send([
            $this->users->userEmail => $this->users->userName
       ], "Reset your password", 'resetpassword', [
           'resetUrl' => '/reset-password/' . $this->resetCode . '/' . $this->users->userEmail
       ]);

```
view loads as expected.

**Comment by sergeyklay** [comment]
@assadnazar What version/branch do you use?

**Comment by assadnazar** [comment]
> @assadnazar What version/branch do you use?

master branch, but a year old download.

**Comment by assadnazar** [comment]
> > @assadnazar What version/branch do you use?
> 
> master branch, but a year old download.

btw, in Mail.php library -> getTemplate() function

why is it returning multiple times (if you can explain) 

```
return $this->view->getRender('emailTemplates', $name, $parameters, function ($view) {
            $view->setRenderLevel(View::LEVEL_LAYOUT);
        });
        return $view->getContent();
```

and is the 'View::LEVEL_LAYOUT' the issue as if i get current level after setting the flash message in 'forgotpassword' action i.e.

```
if ($resetPassword->save())
                        {
                            $this->flash->success('Success! Please check your messages for an email reset password.');
                        }
```

i get 0.

Any idea?

**Comment by Jeckerson** [comment]
@assadnazar Yes. Already fixed in `4.0` branch.

You can find fixes here:
* https://github.com/phalcon/vokuro/pull/122/files#diff-085383d66be26b2b350d584ea4e566aaR59
* https://github.com/phalcon/vokuro/pull/122/files#diff-4ba605a90b065121f21f9322290be780R138

**Comment by assadnazar** [comment]
> @assadnazar Yes. Already fixed in `4.0` branch.
> 
> You can find fixes here:
> 
> * https://github.com/phalcon/vokuro/pull/122/files#diff-085383d66be26b2b350d584ea4e566aaR59
> * https://github.com/phalcon/vokuro/pull/122/files#diff-4ba605a90b065121f21f9322290be780R138
> 
> The problem was in 2 things:
> 
> 1. email was sent during each update of user
> 2. activation of user wasn't working

I dont think so that Reset Password Model does anything to users model. In my query, I was asking about ResetPassword Model.

**Comment by Jeckerson** [comment]
Feel free to re-open, if you still have questions or issues with `v2.0.0`.
